src/main/java/stsjorbsmod/cards/cull/deckoftrials/DeckOfTrials.java

```diff
@@ -0,0 +1,42 @@
+package stsjorbsmod.cards.cull.deckoftrials;
+
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardAndObtainEffect;
+import stsjorbsmod.relics.BookOfTrialsRelic;
+
+import java.util.ArrayList;
+
+public class DeckOfTrials {
+    public static CardGroup deck = new CardGroup(CardGroup.CardGroupType.UNSPECIFIED);
+
+    public static void reset() {
+        deck.group.clear();
+        deck.addToTop(new Frostbite());
+        deck.shuffle();
+    }
+
+    public static ArrayList<AbstractCard> drawCards(int num) {
+        ArrayList<AbstractCard> cards = new ArrayList<>();
+        for (int i = 0; i < num; i++) {
+            if (!deck.isEmpty()) {
+                AbstractCard card = deck.getTopCard();
+                deck.removeCard(card);
```

Comment:
We'll probably want to add a CustomSaveable implementation for this deck state, to avoid players being able to get duplicate trials cards after a save/load.

Reply:
(ideally we'd want to do a specific saveable for this rather than relying on enumerating the player's master deck on load, since players can remove cards)

Reply:
As it stands, it has a different issue of not reverting to the original deck state on save/load. Since the object is static, it stays in memory and on save/load after entering a new act, the deck of trials contains two fewer cards.

Reply:
And because of the way the deck is instantiated, on loading the run after killing STS, the deck of trials is completely empty